### ISSUE TITLE
killed for time vs killed for memory?

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#3147]
When student code is terminated for time or for memory students see a message about the program terminated or killed or kill signal, but they don't know why.

Let's be explicit, was it time or memory?
And also, let's put this in plain english so newbie programmers understand.

[note: this is a duplicate of earlier issue]

**Comment by johnnyapol** [comment]
Would you be in favor in doing this via adding our own syscall or modifying OutputSignalErrorMessageToExecuteLogfile to take in an optional data flag that would be used with SIGKILL is called to differentiate if it was killed for time or memory?

**Comment by bmcutler** [comment]
I'm hoping this is just changing the logging/messages back to the user.  It's been a while since I've been in the code so I'm not sure what specific changes are needed (and if it's simple or going to be messy with lots of changes/refactoring).
